src/problem_bank_helpers/problem_bank_helpers.py

```diff
@@ -382,11 +382,49 @@ def base64_decode_file(file):
     return base64.b64decode(to_decode.encode("utf-8")).decode("utf-8")
 
 def string_to_pl_user_file(string, data):
-    """Encode a string to base64 and add it as the user submitted code file  
+    """Encode a string to base64 and add it as the user submitted code file
     """
     # partially based off of https://github.com/PrairieLearn/PrairieLearn/blob/2ff7c5cc2435bae80c0ba512631749f9c3eadb43/apps/prairielearn/elements/pl-file-upload/pl-file-upload.py#L114C1-L119
     parsed_file = {"name": "user_code.py", "contents": base64_encode_string(answer)}
     if isinstance(data["submitted_answers"].get("_files", None), list):
         data["submitted_answers"]["_files"].append(parsed_file)
     else:
         data["submitted_answers"]["_files"] = [parsed_file]
+
+def convert_markdown_table(table, width="100%", first_row_is_header=True, first_col_is_header=True, wrap_latex=False, wrap_header_latex=False):
```

Comment:
I would change the name of this function and rename the `wrap_latex` argument, and reorder the last two arguments.

```suggestion
def create_html_table(table, width="100%", first_row_is_header=True, first_col_is_header=True, wrap_header_latex=False, wrap_nonheader_latex=False, ):
```